config/schemas/config.yaml

```diff
@@ -1923,916 +3125,1221 @@ properties:
+            title: Grafana Enabled
             type: boolean
-          nodeSelector:
-            $ref: '#/$defs/kubernetesNodeSelector'
+            default: true
+          subdomain:
+            title: Grafana Subdomain
+            description: |-
+              For Admin Grafana the subdomain of `opsDomain` that the Ingress to Admin Grafana will be created with.
+
+              For Dev Grafana the subdomain of `baseDomain` that the Ingress to Dev Grafana will be created with.
+
+              > [!note]
+              > Must be set for both service and workload clusters.
+            default: grafana
+            type: string
           oidc:
+            title: Grafana OIDC
+            description: Configure authentication to Grafana via Dex.
+            type: object
             additionalProperties: false
             properties:
+              enabled:
+                title: Grafana OIDC Enabled
+                type: boolean
+                default: true
               allowedDomains:
+                title: Grafana OIDC Allowed Domains
+                description: Configure the domains of the users allowed to authenticate to Grafana.
                 items:
+                  title: Grafana OIDC Allowed Domain
                   type: string
                 type: array
-              enabled:
-                default: true
-                type: boolean
               scopes:
-                default: openid profile email groups
+                title: Grafana OIDC Scopes
                 type: string
+                default: openid profile email groups
+              skipRoleSync:
+                title: Grafana Skip Role Sync
+                description: When enabled the roles for user can be managed within Grafana.
+                type: boolean
               userGroups:
+                title: Grafana OIDC Groups
+                description: Configure the roles for groups.
+                type: object
                 additionalProperties: false
                 properties:
                   grafanaAdmin:
-                    default: grafana_admin
+                    title: Grafana OIDC Admin Group
                     type: string
+                    default: grafana_admin
                   grafanaEditor:
-                    default: grafana_editor
+                    title: Grafana OIDC Editor Group
                     type: string
+                    default: grafana_editor
                   grafanaViewer:
-                    default: grafana_viewer
+                    title: Grafana OIDC Viewer Group
                     type: string
-                type: object
-              skipRoleSync:
-                type: boolean
+                    default: grafana_viewer
+          viewersCanEdit:
+            title: Grafana Viewers Can Edit
+            type: boolean
+            default: true
+          additionalConfigValues:
+            title: Grafana Additional Config Values
+            type: object
```

Comment:
Seems to be a string and not an object 
https://github.com/elastisys/compliantkubernetes-apps/blob/a2d9ee1823a7bb8c67d6697739324a71e4060061/config/config/sc-config.yaml#L158-L161